action.yml

```diff
@@ -11,6 +11,10 @@ inputs:
   email:
     description: "The GitHub user's email address"
     required: true
+  semver:
+    description: "Use semantic versioning instead of a single number"
+    required: false
+    default: "false"
```

Comment:
Is it necessary to set this to false here as well? Seems to be set to default to false everywhere else in the code as well.

Reply:
So I wanted to have the variable defined at all times. The GH documentation is not very explicit about the behavior, but it looked that if the variable is not provided and has no default value defined, it won't be set. I also wanted to make the default value explicit (also for documentation purposes) in the action YAML.

Technically, the change should work just fine even without the default value being set.

Reply:
Right, I thought I'd rather have it defined in the one place where it actually matters. Maybe it's enough to define it here and not in `argparser`?

Reply:
Not really. Defining it in the script as well makes it possible to run the script locally, which I did for testing. So one could say that the action and the script have their own default, which happens to be the same. But you can't have it only in one place if you want to allow users to run the script outside of the action, which I find useful for testing.

Also not having the default defined in the `argparser` will mean that the key won't be available in the parsed arguments if the option was not provided. That then implies checking if the key exists at all, etc... I find defining the default easier to follow.